_data/menu_docs_09.json

```diff
@@ -1035,6 +1035,10 @@
             {
               "page": "TPC-H",
               "url": "tpch"
+            },
+            {
+              "page": "VSS",
+              "url": "vss"
```

Comment:
Unsure where this is used, I would assume for older versions (where this is not available).
Minor either way.